bears/markdown/MarkdownBear.py

```diff
@@ -101,7 +100,11 @@ def create_arguments(filename, file, config_file,
             'ruleRepetition': markdown_horizontal_rule_repeat,  # int
         }
 
-        config_json = json.dumps(remark_configs)
-        # Remove { and } as remark adds them on its own
-        settings = config_json[1:-1]
-        return '--no-color', '--quiet', '--setting', settings
+        return json.dumps(remark_configs)
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        if config_file:
+            return '--no-color', '--quiet', '--setting', config_file
+        else:
+            return (filename,)
```

Comment:
shouldn't we rather throw like an exception in this case? There should always be a config generated